conjure-java-core/src/integrationInput/java/com/palantir/product/PrimitiveStrictExample.java

```diff
@@ -164,14 +173,32 @@ public Builder ints(@Nonnull Iterable<Integer> ints) {
         }
 
         @Override
-        @JsonSetter(value = "doubles", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
+        @JsonSetter(value = "ints", nulls = Nulls.SKIP, contentNulls = Nulls.FAIL)
+        public Builder ints(@Nonnull int... ints) {
+            checkNotBuilt();
+            this.ints = ConjureCollections.newNonNullIntegerList();
```

Comment:
Interested in everyone's take on whether this level of defensiveness is strictly (heh) necessary. Anyone that is determined enough to break out of the type safety of the strict staged builder kinda DGAF anymore. I only ask because factoring to support this added a ton of `strict` sprawl.

Reply:
> Anyone that is determined enough to break out of the type safety of the strict staged builder kinda DGAF anymore

I think there's a bit of subtlety here because it's not only misuse of the strict builder that could trigger the case you're defending against, but also deserialization where we tell jackson `@JsonDeserialize(builder = PrimitiveStrictExample.DefaultBuilder.class)`. It's entirely reflective and doesn't know how to use staged builders, so the package-private implementation is used directly.

I'm supportive of being defensive here, I generally weigh the quality of the codegen output much higher than the quality of the codegen implementation code -- not that it should be _bad_, but I'd prefer to make the generated code as close to perfect as we can reasonably manage.

Reply:
There's an argument for adding an overload of `public static List<Integer> newNonNullIntegerList(Iterable<Integer> iterable)` to do this in one method call `public static List<Integer> newNonNullIntegerList(int[] items)` and simplify the generated code a bit, but that would take a bit to propagate out. I don't think the upside is necessarily high enough to justify broadening the API and managing its rollout.

Reply:
I am likely gonna have to add another overloaded endpoint for something akin to `public static void addToDoubleList(double double)` for an optimized path to singular adds. I could throw this in as well and just allow it to bake. Im not in a rush to get this in. But I did want it out of my head.

Reply:
https://github.com/palantir/conjure-java/pull/2475

Reply:
I rebased on ^ so now this is a non-issue 👍🏼